app/lab/components/testOrderResultItemForm.html

```diff
@@ -0,0 +1,37 @@
+<form class="item-form" name="form" ng-submit="vm.onSubmit({item: vm.item, result: vm.$item.value});">
+
+  <div class="form-group" ng-class="{'has-error': vm.hasFocus && form.$dirty && !form.$valid}">
+    <label class="col-sm-4" for="value">{{vm.$item.testOrder.concept.display}}</label>
```

Comment:
Should display the same labels used in [RDE](http://dev.esaude.org/openmrs/module/htmlformentry/htmlForm.form?id=8), a couple of options:
(1) Map the concepts with custom labels in POC i18n (Preferable)
(2) Rename the concept to use the same RDE form label